src/index.html

```diff
@@ -2,13 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Title</title>
+    <title>Tree</title>
     <link
       rel="stylesheet"
       href="./styles/main.scss"
     />
   </head>
   <body>
-    <script src="scripts/main.js"></script>
+    <div><ul id="tree"></ul></div>
```

Comment:
```suggestion
    <div id="tree"></div>
```

enough to create div with id tree.